tiltak/src/main/kotlin/no/nav/amt/tiltak/deltaker/service/BrukerServiceImpl.kt

```diff
@@ -185,58 +181,23 @@ class BrukerServiceImpl(
 		log.info("Brukere oppdatert. Det var $brukereOppdatert oppdateringer.")
 	}
 
-	private fun createBruker(fodselsnummer: String): BrukerDbo {
-		val tildeltVeilederNavIdent = personService.hentTildeltVeilederNavIdent(fodselsnummer)
-
-		val veileder = tildeltVeilederNavIdent?.let { navAnsattService.getNavAnsatt(it) }
-
-		val navEnhet = navEnhetService.getNavEnhetForBruker(fodselsnummer)
-
-		val personKontaktinformasjon = personService.hentPersonKontaktinformasjon(fodselsnummer)
-
-		val person = personService.hentPerson(fodselsnummer)
-
-		val erSkjermet = personService.erSkjermet(fodselsnummer)
-
-		val bruker = BrukerUpsertDbo(
-			personIdent = fodselsnummer,
-			fornavn = person.fornavn,
-			mellomnavn = person.mellomnavn,
-			etternavn = person.etternavn,
-			telefonnummer = person.telefonnummer ?: personKontaktinformasjon.telefonnummer,
-			epost = personKontaktinformasjon.epost,
-			ansvarligVeilederId = veileder?.id,
-			navEnhetId = navEnhet?.id,
-			erSkjermet = erSkjermet
-		)
+	override fun upsert(bruker: Bruker) {
+		brukerRepository.upsert(bruker)
+	}
 
-		val brukerDbo = brukerRepository.upsert(bruker)
+	override fun erAdressebeskyttet(personident: String) = amtPersonClient.hentAdressebeskyttelse(personident)
```

Comment:
nit: Tenker litt på navngivningen her, at det fremstår utenfra som at dette er noe man henter fra databasen men så skjer det et restkall. Kan være en fordel at denne også heter `hentErAddresseBeskyttet`

Reply:
Hmm, jeg vet ikke om jeg liker `hentErAdresseBeskyttet` det ruller ikke lett av tunga 😅 og vi bruker `hent`-prefikset ofte når vi bare gjør databasekall også så jeg tror ikke det gjør det så mye mer tydelig at det er et restkall.

Kanskje vi bare kan bruke `AmtPersonClient` direkte der det er behov for å vite om adressebeskyttelsen istedet?